src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -766,6 +783,9 @@ private static ClassLoader contextClassLoader(Thread parent) {
             this.contextClassLoader = ClassLoader.getSystemClassLoader();
         }
 
+        // special value to mean a new thread
+        this.scopedValueBindings = Thread.class;
```

Comment:
The addition of NEW_THREAD_BINDINGS means this one should change too. The update means the comment should probably be adjusted too, maybe  "initial value for a new thread".

Reply:
Done